documentation/modules/adding-source-provider.adoc

```diff
@@ -100,7 +100,7 @@ ifdef::rhv[]
 * *Provider resource name*: Name of the source provider
 +
 include::snip-provider-names.adoc[]
-* *URL*: API endpoint of the {rhv-short} Manager on which the source VM is mounted
+* *URL*: API endpoint of the {rhv-full} Manager (RHVM) on which the source VM is mounted, for example, `https://rhv-host-example.com/ovirt-engine/api`
```

Comment:
in https://github.com/kubev2v/forklift-console-plugin/pull/744/files#diff-075b2af748cc5c8407a5cd156371011523517ef954d89d2bac7d9d5c72d9fe97R29

we chose to write:

> URL of the Red Hat Virtualization Manager (RHVM) API endpoint. Ensure the URL includes the "/ovirt-engine/api" path. for example: https://rhv-host-example.com/ovirt-engine/api

I'm ok with current version and ok with uipdateing to match to UI

Reply:
p.s. not that the `for` -> `For` was fixed in the UI in a folowup

Reply:
@yaacov I added "URL of the..." to all the URL descriptions.